LiteDB.FSharp.Tests/Tests.InheritedType.fs

```diff
@@ -100,7 +101,7 @@ type Item2OfRecord =
 
 
 let useDatabase (f: LiteRepository -> unit) = 
-    let mapper = FSharpBsonMapper()
+    let mapper = new TypeShapeMapper()
```

Comment:
The idea is not to replace but to let the same tests run for both mappers

Reply:
Refactoring made, my mistake, too early push